app/src/main/res/values-fr/strings.xml

```diff
@@ -167,12 +169,14 @@
     <string name="module_release_load_more">Afficher les anciennes versions</string>
     <string name="module_release_no_more">Pas d\'autres versions</string>
     <string name="repo_load_failed">Échec de chargement du dépôt des modules: %s</string>
-
+  
+    <string name="about_source" translatable="false">https://github.com/LSPosed/LSPosed/</string>
 
     <string name="app_destroyed">Cette application à été détruite, merci de vous assurer que vous l\'avez bien téléchargé depuis la source officielle.</string>
     <string name="outdated_manager">Version du Gestionnaire LSPosed et du noyau LSPosed ne sont pas cohérents. Merci de réinstaller la version correspondante.</string>
     <string name="lsposed_not_active">LSPosed n\'est pas installé ou n\'est pas actif.</string>
     <string name="verbose_log_not_avaliable">Journal détaillé indisponible, si vous l\'avez juste activé sans redémarrer, essayez à nouveau après avoir redémarré.</string>
     <string name="NotInstall">Non installé</string>
     <string name="NotInstallDetail">LSPosed n\'est pas installé</string>
+    <string name="translators"><![CDATA[<a href="https://github.com/LSPosed/LSPosed">LSPosed</a>]]></string>
```

Comment:
Add your name to here

Reply:
I have to add my Nickname instead of the URL ??

Reply:
<![CDATA[Your name here]]>

Reply:
URL is ok

Reply:
so:
`<string name="translators"><![CDATA[<a href="https://github.com/LSPosed/LSPosed">MY-NAME</a>]]></string>`
?

Reply:
change link to your website or just remove whole <a > tag and write

Reply:
Ah ok ok